statmoments/_native.pyx

```diff
@@ -673,9 +688,11 @@ class bivar_cntr(_AccBase):
 
       ar_len = tr_layout.shape[1] * tr_layout.shape[2]
       if n0 > 0:
-        dgemm(tr_layout[:n0].reshape((-1, ar_len)), tr_layout[:n0].reshape((-1, ar_len)), self._accs2d[i, 0].reshape(ar_len, ar_len), transa=b'N', transb=b'T')
+        dgemm(tr_layout[:n0].reshape((-1, ar_len)), tr_layout[:n0].reshape((-1, ar_len)),
```

Comment:
let's put all data on one line and params like trans to another line. Now they are mixed up.